GiniVision/Classes/Camera.swift

```diff
@@ -66,10 +66,11 @@ internal class Camera {
                 return completion({ _ in throw CameraError.noInputDevice })
             }
             // Set the orientation accoding to the current orientation of the device
+            
             if let orientation = AVCaptureVideoOrientation(self.motionManager.currentOrientation) {
                 connection.videoOrientation = orientation
             } else {
-                connection.videoOrientation = .portrait
+                connection.videoOrientation = AVCaptureVideoOrientation(UIApplication.shared.statusBarOrientation)
```

Comment:
I found out that when the device is on flat position (neither landscape or portrait) there is no UIDeviceOrientation, and in that case it would not be correct to fallback always to portrait, given that it is not correct and we have another orientation reference (UIInterfaceOrientation).